packages/syft/src/syft/core/tensor/autodp/phi_tensor.py

```diff
@@ -2022,6 +2025,21 @@ def sum(
         #     max_vals=max_val,
         # )
 
+    def mean(
+        self,
+        axis: Optional[Union[int, Tuple[int, ...]]] = None,
+        **kwargs: Any,
+    ) -> PhiTensor:
+        child = self.child.mean(axis=axis, **kwargs)
+        data_subjects = np.array(self.data_subjects.mean(axis=axis, **kwargs))
+
+        return PhiTensor(
+            child=child,
+            data_subjects=data_subjects,
+            min_vals=self.min_vals.mean(),
```

Comment:
Pass in args and kwargs so that the min_vals and max_vals have the same shape as child if the Data Scientist uses any args/kwargs